dev/publicDev/emergencyTowing/service.js

```diff
@@ -0,0 +1,15 @@
+'use strict';
+
+service.$inject = ['$http', 'TokenService'];
+function emergencyTowingService ($http, TokenService) {
+  return {
+    getEmergencyTowing: function() {
```

Comment:
Move to "dev\publicDev\photos\dataProvider.js" and rename "dev\publicDev\photos\dataProvider.js" service please (wrong name "propertiesService").